src/main/java/edu/cornell/kfs/pdp/document/authorization/CuPayeeACHAccountMaintenanceDocumentAuthorizer.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.pdp.document.authorization;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.kim.api.identity.Person;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.pdp.document.authorization.PayeeACHAccountMaintenanceDocumentAuthorizer;
+
+public class CuPayeeACHAccountMaintenanceDocumentAuthorizer extends PayeeACHAccountMaintenanceDocumentAuthorizer {
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public boolean canApprove(Document document, Person user) {
+
+        try {
+            String documentInitializerPrincipalId = document.getDocumentHeader().getWorkflowDocument().getInitiatorPrincipalId();
+            if (user.getPrincipalId().equals(documentInitializerPrincipalId)) {
+                return false;
```

Comment:
As per the user story's requirements, there are certain initiators that may still need to be allowed to approve the document, such as the KFS system user and those with superuser privileges (and possibly those with blanket-approve privileges). Please adjust this code accordingly to account for that.

Reply:
See the other comments on my latest review for further details on the can-approve setup.